code/style.css

```diff
@@ -1,5 +1,315 @@
+}
+
+p {
+  font-size: 14px;
+  margin: 0;
+}
+
+/*** HEADER ***/
+
+header {
+  background-color: rgb(66, 116, 113);
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 20px;
+  border-radius: 20px;
+  font-size: 18px;
+}
+
+header img {
+  height: 100%;
+}
+
+nav a {
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  margin-right: 16px;
+}
+
+/*** INTRODUCTION WORK ***/
+
+.Introduction-Work {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px 16px;
 }
+ .Introduction-Work img {
+   height: 100%;
+   width: 100%;
+   max-width: 1100px;
+ }
+ .Introduction-Work div {
+   color: rgb(245, 255, 255);
+   text-align: center;
+   margin-bottom: 8px;
+ }
+
+ .mini-menu-links {
+   display: flex;
+   align-items: center;
+   justify-content: space-evenly;
+   align-items: center;
+   margin-bottom: 32px;
+   margin-top: 24px;
+ }
+
+ .mini-menu-links div {
+  width: 3px;
+  height: 32px;
+  background: white;
+  border-radius: 5px;
+  margin-bottom: 0px;
+ }
+
+ .mini-menu-links a {
+   text-decoration: none;
+   color: white;
+ }
+
+ /*** ANIMATIONS ***/
+
+ .Animations {
+   display: flex;
+   flex-wrap: wrap;
+   justify-content: center;
+   width: 100%;
+ }
+
+ .Animations-Titel img {
+   height: 120px;
+   width: 120px;
+   object-fit: cover;
+   border-radius: 20%;
+ }
+
+ .Animations-Titel {
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   padding: 32px 32px;
+   margin-bottom: 16px;
+ }
+
+ .Animations-Titel div {
+   text-align: center;
+  }
+
+
+/** Backseat Card **/
+
+ .Backseat-Card {
+   background-color: rgb(72, 100, 104);
+   color: rgb(255, 255, 255);
+   margin-bottom: 32px;
+   border-radius: 25px;
+   padding: 16px;
+   max-width: 650px;
+ }
+
+ .Backseat-Card div p{
+   padding: 5px 16px 8px;
+ }
+
+.Backseat-trailer-link {
+  display: flex;
+  flex-direction: column;
+  padding: 0%;
+  position: relative;
+}
+
+.Backseat-trailer-link .btn {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  background-color:rgb(35, 35, 94);
+  color: rgb(194, 194, 224);
+  opacity: 80%;
+  font-size: 16px;
+  padding: 16px 30px;
+  border: none;
+  cursor: pointer;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.Backseat-trailer-link .btn:hover {
+  background-color: rgb(78, 34, 34);
+  color: white;
+  opacity: 100%;
+}
+
+.Backseat-trailer-link a img {
+  width: 100%;
+  margin-top: 0;
+}
+
+/** Another world card **/ 
+
+.another-world-card {
+    background-color: rgb(72, 100, 104);
+    color: rgb(255, 255, 255);
+    border-radius: 25px;
+    padding: 16px;
+    max-width: 650px;
+  }
+ 
+  .another-world-card div p{
+    padding: 5px 16px 8px;
+  }
+ 
+  .another-world-trailer-link {
+    display: flex;
+    flex-direction: column;
+    padding: 0%;
+    position: relative;
+  }
+  
+  .another-world-trailer-link .btn {
```

Comment:
To use the same styling as the other button, you could write like this instead to minimize amount of code: 
`
.Backseat-trailer-link .btn, .another-world-trailer-link .btn {
... styling in here...
}
`